content/webapp/views/components/RelatedWorksCard/RelatedWorksCard.Hover.tsx

```diff
@@ -0,0 +1,79 @@
+import { FunctionComponent } from 'react';
+
+import { convertIiifImageUri } from '@weco/common/utils/convert-image-uri';
+import { DataGtmProps, dataGtmPropsToAttributes } from '@weco/common/utils/gtm';
+import { WorkBasic } from '@weco/content/services/wellcome/catalogue/types';
+import { ContentApiLinkedWork } from '@weco/content/services/wellcome/content/types/api';
+
+import {
+  Card,
+  ImageWrapper,
+  LineClamp,
+  MetaContainer,
+  TextWrapper,
+  Title,
+} from './RelatedWorksCard.styles';
+
+type Props = {
+  work: WorkBasic | ContentApiLinkedWork; // Supports both Catalogue and Content API works
+  gtmData?: DataGtmProps;
+};
+
+const RelatedWorksCard: FunctionComponent<Props> = ({ work, gtmData }) => {
+  const isCatalogueWork = 'notes' in work;
+
+  const thumbnailUrl = isCatalogueWork
+    ? work.thumbnail?.url
+    : work.thumbnailUrl;
+
+  const date = isCatalogueWork
+    ? work.productionDates.length > 0
+      ? work.productionDates[0]
+      : undefined
+    : work.date;
+
+  const mainContributor = isCatalogueWork
+    ? work.primaryContributorLabel
+    : work.mainContributor;
```

Comment:
I think if both card types use the same data, these could be centralised to not be "transformed" twice/ease future maintenance/changes.